solutions/testing/.github/workflows/playwright.yml

```diff
@@ -1,27 +1,25 @@
-name: Playwright Tests
+name: Playwright
+
 on:
-  push:
-    branches: [main, master]
```

Comment:
Will it run on all branches now? This should only fire when I open a PR and on the main branch

Reply:
On every deployment, the code that existed before was automatically generated by Playwright when I was creating the example, it was all wrong.